vars/newtestpipeline.groovy

```diff
@@ -0,0 +1,127 @@
+#!groovy
```

Comment:
please don't duplicate this file, please add it to the longevity one.
it's safe enough as long you are on your branch testing it.

Reply:
fixed